main.tex

```diff
@@ -195,20 +204,14 @@
 
 \item  install and operate on the Common Property closed-circuit television monitoring devices, audio- visual cameras and other audio-visual surveillance devices to monitor the security, safety and use of Common Property and Lots;
 
-\item  restrict by means of Security Keys, the access of Occupiers to any Common Property;
+\item  require the use of Security Keys for Occupiers to access any Common Property;
 
 \item  restrict by means of Security Keys, the access of Occupiers of one level of the Development to any other level of the Development;
 \newpage
 
+\item  control Security Keys by publicised time periods and enforce publicised use-by dates;
 
-
-\item  control Security Keys by time periods and enforce use by dates;
-
-\item  the right (upon complaint) to remove any person from the Common Property or to refuse admission to any person it considers likely to be a nuisance or a security risk;
```

Comment:
Outrageous overreach. Who is a "nuisance"? The Owners Corporation currently has the power to ban someone from Common Property merely because the Owners Corporation doesn't like them.